src/server.ts

```diff
@@ -1,44 +1,36 @@
-import {
-  server as WebSocketServer, // these are NOT the correct typings!
-  connection as Connection,
-  request as WebSocketRequest,
-  IMessage,
-} from 'websocket';
+import * as WebSocket from 'ws';
 
 import {
   SUBSCRIPTION_FAIL,
   SUBSCRIPTION_DATA,
   SUBSCRIPTION_START,
   SUBSCRIPTION_END,
   SUBSCRIPTION_SUCCESS,
-  SUBSCRIPTION_KEEPALIVE,
+  SUBSCRIPTION_KEEPALIVE, INIT, INIT_FAIL, INIT_SUCCESS,
 } from './messageTypes';
-import { GRAPHQL_SUBSCRIPTIONS } from './protocols';
+import {GRAPHQL_SUBSCRIPTIONS} from './protocols';
 
-import { SubscriptionManager } from 'graphql-subscriptions';
-import { SubscriptionOptions } from 'graphql-subscriptions/dist/pubsub';
-import { Server as HttpServer} from 'http';
+import {SubscriptionManager} from 'graphql-subscriptions';
+import {SubscriptionOptions} from 'graphql-subscriptions/dist/pubsub';
```

Comment:
Importing from `dist` seems super wrong.  I think graphql-subscriptions should probably be exporting more things?